infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/KubernetesServerResolverTest.java

```diff
@@ -263,4 +265,10 @@ private Ingress createIngress(
         .endStatus()
         .build();
   }
+
+  private Map<String, String> defaultAttributeAnd(String key, String value) {
+    HashMap<String, String> map = new HashMap<>(ATTRIBUTES_MAP);
+    map.put(key, value);
+    return map;
```

Comment:
It's a bad manner to name variable after its type

Reply:
ok

Reply:
I mean I'll rename it